.changeset/dull-jobs-tap.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/prettier-plugin-liquid': minor
+'@shopify/liquid-html-parser': minor
+---
+
+updated theme-tools ohm rules to accept snippet tag
```

Comment:
Given this shows up in our changelogs, I'd make it look a little better :D 

Something like: Add parsing support for the `snippet` tag